Project.toml

```diff
@@ -21,7 +21,7 @@ SBML = "1.4.4"
 SBMLToolkitTestSuite = "0.0.3"
 SafeTestsets = "0.1"
 Sundials = "4.14"
-SymbolicUtils = "2"
+SymbolicUtils = "1, 2"
```

Comment:
why would we add back 1?

Reply:
I thought the less restrictive we are with our compats, the less likely developers who may depend on SBMLtk have compatibility issues. Also because of a [compatibility requirement](https://github.com/SciML/Catalyst.jl/blob/7a536673f1d8140535db1aa6cd5f66b6d4c44d25/Project.toml#L43) with the latest Catalyst release. Looks like the next release will no longer have a SymbolicUtils requirement, tho.